vmm/src/device_manager.rs

```diff
@@ -591,6 +591,7 @@ impl DeviceManager {
 
```

Comment:
This commit message is too long:

`vmm: device_manger: Make serial and ioapic devices migratable`

Reply:
Ok!